core/api/src/app/wallets/update-single-pending-invoice.ts

```diff
@@ -348,21 +363,31 @@ const lockedUpdatePendingInvoiceSteps = async ({
     additionalInternalMetadata: internalAccountsAdditionalMetadata,
   })
   if (journal instanceof Error) {
-    return ProcessPendingInvoiceResult.paidWithError(journal)
+    recordExceptionInCurrentSpan({
+      error: journal,
+      level: ErrorLevel.Critical,
+      attributes: {
+        ["error.actionRequired.message"]:
+          "Check that invoice exists and is settled in lnd, unmark paid (and processed) from ledger entries",
+        ["error.actionRequired.paymentHash"]: paymentHash,
+        ["error.actionRequired.pubkey]"]: walletInvoiceInsideLock.pubkey,
+      },
+    })
+    return ProcessPendingInvoiceResult.err(journal)
```

Comment:
I have my doubts about this.. what if the error is in our side and not in db? should we include validation against medici txs too?

Reply:
Message changed inside this [`force-push`](https://github.com/GaloyMoney/galoy/compare/cdcfcd764263b83477fb6f59b707d69ec4571f1e..ee294b7bbe2a5c9a9a8b335576a15389f9de28c7)